src/components/BookDetail.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+function BookDetail() {
+  const books = useSelector((state) => state.books);
+
+  const { name } = useParams();
+  const findBook = books.books.find((book) => book.name === name);
+
+  if (!findBook) {
+    return <h2>Book not found</h2>;
+  }
+
+  return (
+    <>
+      <div className="main">
+        <p className="detail-title">Book Detail page</p>
+        <div className="book-detail">
+          <div className="book-image">
+            <img className="img" src={findBook.image} alt={findBook.name} />
+          </div>
+          <div className="book-details">
+            <h3>{findBook.name}</h3>
+            <table className="styled-table">
+              <tbody>
+                <tr>
+                  <th scope="row">Author</th>
```

Comment:
Be sure to also add the `info` property that is returned from getting books in this page!
![image](https://github.com/leloufadel/front-end-capstone/assets/91759364/a18cb607-ec39-4afd-9052-c3ff5541d8b6)